### ISSUE TITLE
Migrate to new toolbar api

### ISSUE THREAD

**Issue by medzhidov** [kaaaaaaaaaaai/editorjs-alignment-blocktune#10]
I migrate this plugin to new api, you just need to make a build of it (i just changed sources) and migrate with master

https://github.com/medzhidov/editorjs-alignment-blocktune/tree/migrate-to-new-api

**Comment by SalahAdDin** [comment]
@medzhidov did you make a pull request for this?

Could you [include](https://github.com/kaaaaaaaaaaai/editorjs-alignment-blocktune/issues/9) this feature?

**Comment by medzhidov** [comment]
@SalahAdDin nope, just created a new repository. I hope maintainer will build it and update master repo. I just coded it for myself and just shared it to repo's owner.

**Comment by SalahAdDin** [comment]
> @SalahAdDin nope, just created a new repository. I hope the maintainer will build it and update the master repo. I just coded it for myself and just shared it with the repo's owner.

it would be easier if you just create a pull request, so you can contribute to this project.

**Comment by medzhidov** [comment]
@SalahAdDin owner of this project was here a year ago, I think this repo is abandoned

**Comment by SalahAdDin** [comment]
> @SalahAdDin owner of this project was here a year ago, I think this repo is abandoned

He is still maintaining some other repos.
